src/main/java/model/Rank.java

```diff
@@ -4,19 +4,21 @@
 import java.util.Optional;
 
 public enum Rank {
-	FIRST(2_000_000_000, 6),
-	SECOND(30_000_000, 5),
-	THIRD(1_500_000, 5),
-	FOURTH(50_000, 4),
-	FIFTH(5_000, 3),
-	NONE(0, 0);
+	FIRST(2_000_000_000, 6, false),
```

Comment:
```suggestion
	FIRST(2_000_000_000, 6),
```

```
Rank(int reward, int matchingCount) {
  this(reward, matchingCount, false);
}
```

생성자를 지원해주면 불필요한 중복을 제거할 수 있겠네요 ㅎㅎ